<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AMatParticleConfigurator</class>
 <widget class="QDialog" name="AMatParticleConfigurator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pbUpdateGlobSet">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>60</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...updateGlobSet</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>meV</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledMaxEnergy">
   <property name="geometry">
    <rect>
     <x>343</x>
     <y>117</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbOnlyInRange">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>94</y>
     <width>321</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Only load cross-section data for energy in range:</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cobUnitsForEllastic">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>meV - barns</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>eV - barns</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>keV - barns</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MeV - barns</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="cbAuto">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>195</y>
     <width>361</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Automatically look for elastic cross-section data files</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>215</y>
     <width>491</width>
     <height>121</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="leEndName">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>.txt</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>7</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cross-section file location:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePreName">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>90</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>elastic</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leSeparatorInName">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>92</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mass</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>133</x>
      <y>92</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Element_name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDir">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>30</y>
      <width>386</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbChangeDir">
    <property name="geometry">
     <rect>
      <x>406</x>
      <y>28</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>67</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File name structure:</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="leNatAbundFile">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>22</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>File format: MassElement     Abundancy%</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>1</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Take natural abundancy data from file:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Min:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pbChangeNatAbFile">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Change</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>120</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>meV</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledMinEnergy">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>117</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0.001</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_34">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>64</y>
     <width>160</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Cross-section are loaded as:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Max:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cbAuto</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbOnlyInRange</sender>
   <signal>toggled(bool)</signal>
   <receiver>ledMinEnergy</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbOnlyInRange</sender>
   <signal>toggled(bool)</signal>
   <receiver>ledMaxEnergy</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cobUnitsForEllastic</sender>
   <signal>activated(int)</signal>
   <receiver>pbUpdateGlobSet</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>482</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ledMinEnergy</sender>
   <signal>returnPressed()</signal>
   <receiver>pbUpdateGlobSet</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ledMaxEnergy</sender>
   <signal>editingFinished()</signal>
   <receiver>pbUpdateGlobSet</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>502</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbAuto</sender>
   <signal>clicked()</signal>
   <receiver>pbUpdateGlobSet</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>524</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lePreName</sender>
   <signal>returnPressed()</signal>
   <receiver>pbUpdateGlobSet</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leSeparatorInName</sender>
   <signal>editingFinished()</signal>
   <receiver>pbUpdateGlobSet</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leEndName</sender>
   <signal>editingFinished()</signal>
   <receiver>pbUpdateGlobSet</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>516</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>538</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
